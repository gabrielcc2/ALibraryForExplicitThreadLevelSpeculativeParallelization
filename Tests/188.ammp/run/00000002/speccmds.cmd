-u /home/gabriel/Escritorio/SPEC_CPU2000_CD_2/benchspec/CFP2000/188.ammp/run/00000002
-i ammp.in >ammp.out 2>ammp.err ../00000002/ammp_base.x86_linux
