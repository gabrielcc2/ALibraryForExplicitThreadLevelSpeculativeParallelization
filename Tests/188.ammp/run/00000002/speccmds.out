running commands in speccmds.cmd 1 times
runs started at 1312082671, 290974000, Sat Jul 30 22:54:31 2011
run 1 started at 1312082671, 291101000, Sat Jul 30 22:54:31 2011
child started: 0, 1312082671, 291120000, 2561, '>ammp.out 2>ammp.err ../00000002/ammp_base.x86_linux'
child finished: 0, 1312082681, 232258000, 9, 941138000, 2561, 0
run 1 finished at: 1312082681, 232269000, Sat Jul 30 22:54:41 2011
run 1 elapsed time: 9, 941168000, 9.941168000
runs finished at 1312082681, 232324000, Sat Jul 30 22:54:41 2011
runs elapsed time: 9, 941350000, 9.941350000
